openapi-cli/src/test/java/io/ballerina/openapi/generators/client/ComparedGeneratedFileTests.java

```diff
@@ -95,6 +95,8 @@ public void  openApiToBallerinaCodeGenTestForClient(String yamlFile, String expe
         compareGeneratedSyntaxTreeWithExpectedSyntaxTree(expectedPath, syntaxTree);
     }
 
+
```

Comment:
Additional new line

Reply:
Updated in https://github.com/ballerina-platform/openapi-tools/pull/1328/commits/1c1b409182ccd309a4da54dd57c18e65241dd1bc